src/components/statevector_component/aggregation.py

```diff
@@ -296,6 +304,115 @@ def generate_cluster_pairs(config, sensitivities):
     return sorted(cluster_pairs, key=lambda x: x[0])
 
 
+def get_plumes(month, year):
+    """
+    Description:
+        Scrapes the SRON database for weekly methane plumes, saving each week's data as csv files
+    arguments:
+        month        String : the month (number) for which to select the plumes
+        year         String : the year (number) for which to select the plumes
+    Returns:                pd.Dataframe() : pandas dataframe with all of the plumes detected for that month
+    """
+    sron_url = "https://earth.sron.nl/wp-content/uploads/"     #URL of the SRON database for weekly methane plumes
+    url = sron_url + year + "/" + month.zfill(2)
+    response = requests.get(url)
+    parser = BeautifulSoup(response.content, 'html.parser')
+    plume = pd.DataFrame()
+    for link in parser.find_all('a'):
+        if ('.csv' in link.get('href') and 'SRON_Weekly_Methane_Plumes' in link.get('href')): #filters through all .csv files containing "SRON_Weekly_Methane_Plumes"
+            csvUrl = url + "/" + link.get('href')
+            dates = csvUrl.split("_v")[1]
+            try:
+                rcsv = requests.get(csvUrl, allow_redirects=True)
+                file = f"SRON_{dates}"
+                open(file, 'wb').write(rcsv.content) # downloads all of the plumes from that week in a CSV file in the current directory
+                df = pd.read_csv(file) # reads from the csv file into a pandas dataframe
+                plume = plume.append(df, ignore_index=True)
+            except:
+                print("ERROR FETCHING CSV FILE")
```

Comment:
Update error message to say: `f"Warning: Unable to access data for csv file at {csvUrl}. The file may not exist or there may be a connection problem."`